src/components/AttachmentPicker/index.native.tsx

```diff
@@ -212,7 +212,7 @@ function AttachmentPicker({type = CONST.ATTACHMENT_PICKER_TYPE.FILE, children, s
      * An attachment error dialog when user selected malformed images
      */
     const showImageCorruptionAlert = useCallback(() => {
```

Comment:
NAB: If we replace the wording from Image to Attachment for the translate key, should we change the function name to `showAttachmentCorruptionAlert `?